tests/test_kubernetes_tools.py

```diff
@@ -3905,7 +3905,7 @@ def test_warning_big_bounce():
             job_config.format_kubernetes_app().spec.template.metadata.labels[
                 "paasta.yelp.com/config_sha"
             ]
-            == "config9e0d925d"
+            == "configc22655c7"
```

Comment:
huh, interesting - I wonder what changed here in the underlying config (maybe the ordering in https://github.com/Yelp/paasta/blob/b75e2d554bdd94249d92807d951d69532d85427e/paasta_tools/utils.py#L3699?)